alibi/utils/distance.py

```diff
@@ -243,3 +242,78 @@ def multidim_scaling(d_pair: dict,
         feature_range = new_feature_range
 
     return d_abs_scaled, feature_range
+
+
+def squared_pairwise_distance(x: np.ndarray, y: np.ndarray, a_min: float = 1e-30, a_max: float = 1e30) -> np.ndarray:
```

Comment:
What's the motivation and choice behind `a_min` and `a_max`? I assume `a_min` is to avoid division by zero, but does the specific value of `1e-30` have some significance (e.g. machine precision for `float64` is given by `np.finfo(float).eps` which is `~2.2e-16`).

Similar for `a_max`, are we worried about overflow in downstream applications?

Reply:
Good point ... I actually took them from [detect](https://github.com/SeldonIO/alibi-detect/blob/a0e7082ebb12206fa7f5417cab0353c55169cfa3/alibi_detect/utils/tensorflow/distance.py#L9) and I didn't realize